src/tools/kdb/editor.cpp

```diff
@@ -96,6 +96,12 @@ int EditorCommand::execute (Cmdline const & cl)
 
 	if (cl.strategy == "validate")
 	{
+		if (cl.ns.empty ())
+		{
+			cerr << "Namespace (-N) needs to be specified for strategy 'validate'." << endl;
```

Comment:
For the editor for every strategy a namespace must be given explicitly.

Reply:
I made this distinction because in libelektra/doc/help/kdb-editor.md it is described:
```-N, --namespace=: Specify the namespace to use when writing cascading keys (validation strategy only).```

Reply:
Obsolete due to https://github.com/ElektraInitiative/libelektra/issues/4012.